lib/global_id/locator.rb

```diff
@@ -19,8 +19,11 @@ def locate(gid, options = {})
       end
 
       # Takes an array of GlobalIDs or strings that can be turned into a GlobalIDs.
-      # The GlobalIDs are located using Model.find(array_of_ids), so the models must respond to
-      # that finder signature.
+      # All GlobalIDs must belong to the same app, as they will be located using
```

Comment:
I've made a tiny edit to the README to highlight Global means app wide: https://github.com/rails/globalid/commit/dbe2d08c3e36cdd8f21b1fa36807e2adb8d3c316

So now I'm going to revert my thinking from yesterday and say I don't think we to explain locate_many is per app.
What do you think, @lucasmazza?